build.sbt

```diff
@@ -149,13 +114,13 @@ lazy val metals = project
       "io.github.soc" % "directories" % "5", // for cache location
       "me.xdrop" % "fuzzywuzzy" % "1.1.9", // for workspace/symbol
       "org.fusesource.leveldbjni" % "leveldbjni-all" % "1.8", // for caching classpath index
+      "org.scalameta" %% "lsp4s" % "0.1.0",
```

Comment:
This will effectively introduce a regression wrt the PR merged yesterday that made pid optional during initialization. Can we release a 0.1.1 of lsp4s and depend on it?